#!/usr/bin/env python

# Copyright (C) 2017 Electric Movement Inc.
# All Rights Reserved.

PACKAGE = "hover_controller"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('target', double_t, 0, 'Altitude Target',  0, 0, 100)
gen.add('kp',     double_t, 0, 'P',                0, 0, 100)
gen.add('ki',     double_t, 0, 'I',                0, 0, 100)
gen.add('ki_max', double_t, 0, 'I Max Windup',     0, 0, 100)
gen.add('kd',     double_t, 0, 'D',                0, 0, 100)

exit(gen.generate(PACKAGE, 'hover_controller', 'hover_controller_params'))
